collecting-training-data/README.md

```diff
@@ -1 +1,36 @@
-Moved to https://github.com/luxonis/depthai-experiments/tree/master/gen2-collecting-training-data
\ No newline at end of file
+[中文文档](README.zh-CN.md)
```

Comment:
This script was moved [here](https://github.com/luxonis/depthai-experiments/tree/master/gen2-collecting-training-data) - could you update the README there? It also have a different content as new flags were added